src/utils/validateUserSlippageTolerance.test.ts

```diff
@@ -0,0 +1,15 @@
+import { Percent } from '@uniswap/sdk-core'
+
+import validateUserSlippageTolerance, { SlippageValidationResult } from './validateUserSlippageTolerance'
+
+describe('validateUserSlippageTolerance', () => {
```

Comment:
Can you add the edge cases where it is _equal to_ the minimum/maximum? These seem the most likely to break on a potential future refactor.

Reply:
Done